.github/workflows/windows-build.yaml

```diff
@@ -23,6 +17,20 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      # Cache the Cargo registry
+      - name: Cache Cargo registry
+        uses: actions/cache@v2
+        with:
+          path: ~/.cargo/registry
+          key: ${{ runner.os }}-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
Here's how we're caching for other rust workflows, maybe we can unify ? https://github.com/aptos-labs/aptos-core/blob/main/.github/actions/rust-setup/action.yaml

They're caching the same thing, though the reuse action is different